tests/features/news.feature

```diff
@@ -40,3 +40,19 @@ Feature: News content creation
     And I should see the link "financing"
     And I should see the link "European Patent Office"
     And I should see the AV Portal photo "Euro with miniature figurines" with source "//ec.europa.eu/avservices/avs/files/video6/repository/prod/photo/store/store2/4/P038924-352937.jpg"
+
+  @javascript
+  Scenario: Length limited fields are not allowing characters above the limit configured.
+    Given I am logged in as a user with the "create oe_news content, access content, edit own oe_page content, view published skos concept entities, create av_portal_photo media" permission
+    When I visit "the News creation page"
+    Then I should see the text "Content limited to 150 characters, remaining: 150"
+    And I should see the text "Content limited to 250 characters, remaining: 250"
```

Comment:
Can we contextualize these two steps? We can simply use Behat regions to target those specific fields, ideally we should have:

```gherkin
    Then I should see the text "Content limited to 150 characters, remaining: 150" in the "Title form element"
    And I should see the text "Content limited to 250 characters, remaining: 250" in the "Teaser form element"
```